coalib/bears/Bear.py

```diff
@@ -24,6 +24,26 @@
 from .meta import bearclass
 
 
+def _debugger_setting_value(bear):
+    """
+    Determines the value of ``debugger`` setting.
+
+    :param bear: Bear object.
+    :return:     True if ``debug_bears`` is ``True`` or if bear name specified
+                 in ``debug_bears`` setting match with the bear parameter.
+    """
+    if bear is None:
+        return False
+    if 'debug_bears' not in bear.section:
+        return False
```

Comment:
Can be merged into a single `if`-condition ;)

Reply:
Which two condition you want me to merge if above you said `if bear is None ` is not a valid condition (I will raise a value error in replacement)

Reply:
I have updated till now what I have got